routers/web/user/setting/security_twofa.go

```diff
@@ -146,15 +146,15 @@ func EnrollTwoFactor(ctx *context.Context) {
 	ctx.Data["Title"] = ctx.Tr("settings")
 	ctx.Data["PageIsSettingsSecurity"] = true
 
-	t, err := models.GetTwoFactorByUID(ctx.User.ID)
+	t, err := login.GetTwoFactorByUID(ctx.User.ID)
 	if t != nil {
 		// already enrolled - we should redirect back!
 		log.Warn("Trying to re-enroll %-v in twofa when already enrolled", ctx.User)
 		ctx.Flash.Error(ctx.Tr("setting.twofa_is_enrolled"))
```

Comment:
The i18n is `settings.twofa_is_enrolled` ( the `s`). Although I fixed it on a WIP Enforce 2FA branch, but I am afraid that branch won't be merged soon .....

Reply:
another PR #17150